DESCRIPTION

```diff
@@ -1,8 +1,8 @@
 Package: gsplot
 Type: Package
 Title: Geological Survey Plotting
-Version: 0.6.1
-Date: 2016-05-12
+Version: 0.6.9
+Date: 2016-07-05
 Authors@R: c( person("Jordan", "Read", role = "aut",
     email = "jread@usgs.gov"),
     person("Laura", "DeCicco", role = c("aut","cre"),
```

Comment:
I had just copied this from something originally, probably want to change @jread-usgs or @lindsaycarr to "cre"